installer/install.iss

```diff
@@ -3366,7 +3366,6 @@ begin
         Form.Caption:='Git Uninstall: Removing in-use files';
         Form.ClientWidth:=ScaleX(500);
         Form.ClientHeight:=ScaleY(256);
-        Form.Center;
```

Comment:
Had to remove this due to the following error:

```
Launching Inno Setup compiler ...
Error on line 3369 in C:\git-sdk-32\usr\src\build-extra\installer\install.iss: Column 14:
Unknown identifier 'CENTER'
```

Reply:
Heh, I did the same in dscho/build-extra@674f32017f43d596f9e49c3a390598aea0c92c07...